### PR TITLE
First

### PR DESCRIPTION
**Pull request by LucasSapient**

### CODE REVIEW

**Review by edmondop — CHANGES_REQUESTED** [review]
Can you please add a gitignore specific for python and remove the .pyc files from the codebase?

thanks

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
```
Comment (edmondop) [inline .vscode/settings.json]: What is this?

**Review by edmondop — CHANGES_REQUESTED** [review]
Please remove the empty README.rst

File: .vscode/settings.json (hunk 1 above)
Comment (LucasSapient) [reply .vscode/settings.json]: Should not be there: removed the directory and added to the gitignore

**Comment by LucasSapient** [comment]
I have removed the empty README and would like to request another review

**Review by edmondop — APPROVED** [review]
Sounds good, next step will be to start adding object model for the following concept:
peaks, window, batch, plate. Go hunt around what these things mean for us :D

**Pull request merged**